client/lib/plans/constants.js

```diff
@@ -1002,3 +1002,16 @@ export function getMonthlyPlanByYearly( plan ) {
 			return '';
 	}
 }
+
+export function getYearlyPlanByMonthly( plan ) {
```

Comment:
It's a shame #14761 hasn't landed yet

There are a lot of helper functions that live in this `constants` file which doesn't seem correct. I think it would be better if we tried to keep these functions in a more appropriate place.